src/plugin/duration/index.js

```diff
@@ -79,15 +79,17 @@ class Duration {
     if (typeof input === 'string') {
       const d = input.match(durationRegex)
       if (d) {
-        [,,
+        const [,, ...properties] = d
```

Comment:
it is not advised to use `...` is dayjs cause it will compile to a lot es5 codes to make it work.

Reply:
It is worth mentioning in `CONTRIBUTING.md` as there is now: 
> Day.js is written in ES6.